### PR TITLE
Update doc for CA1716 to mention configurability of symbol kinds

### LINKED ISSUE #4982: CA1716 - Add a new configurability section
For the next release of FxCop (v3), the documentation must be updated to include a new configurability section (see https://github.com/dotnet/roslyn-analyzers/pull/3428/files#diff-1b5cdc4937a8cdfc79545945ad6c0783R505-R514).

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 21d618b6-87c0-6d8e-d29f-b8b35be2900d
* Version Independent ID: 32dcbb65-1464-a1fb-a44d-1afa16a1f4e0
* Content: [CA1716: Identifiers should not match keywords - Visual Studio](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1716?view=vs-2019#feedback)
* Content Source: [docs/code-quality/ca1716.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/code-quality/ca1716.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-code-analysis**
* GitHub Login: @Mikejo5000
* Microsoft Alias: **mikejo**

### PR DESCRIPTION
**Pull request by Evangelink**
Fix #4982

This new feature is not yet publicly released. Please sync-up with @mavasani for when to merge this PR.

### CODE REVIEW

**Comment by PRMerger20** [comment]
@Evangelink : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/code-quality/ca1716.md [hunk 1]
```diff
@@ -52,10 +52,20 @@ You can suppress a warning from this rule if you're convinced that the identifie
 
 ## Configurability
 
+### Analyzed types and members surface
```
Comment (mavasani) [inline docs/code-quality/ca1716.md:55]: Let's be consistent with the heading: https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1802?view=vs-2019#api-surface

```suggestion
### Api Surface
```

File: docs/code-quality/ca1716.md [hunk 2]
```diff
@@ -52,10 +52,20 @@ You can suppress a warning from this rule if you're convinced that the identifie
...
+
 If you're running this rule from [FxCop analyzers](install-fxcop-analyzers.md) (and not with legacy analysis), you can configure which parts of your codebase to run this rule on, based on their accessibility. For example, to specify that the rule should run only against the non-public API surface, add the following key-value pair to an .editorconfig file in your project:
 
 ```ini
 dotnet_code_quality.ca1716.api_surface = private, internal
 ```
 
 You can configure this option for just this rule, for all rules, or for all rules in this category (Naming). For more information, see [Configure FxCop analyzers](configure-fxcop-analyzers.md).
+
+### Analyzed symbol kinds
```
Comment (mavasani) [inline docs/code-quality/ca1716.md:65]: Please add a small sentence about configurability of analyzed symbol kinds at the end of the `Cause` section.

**Comment by ktoliver** [comment]
Hi @Evangelink - Could you make the revisions that @mavasani requests? Then we can merge the PR. Thanks.

**Comment by PRMerger15** [comment]
@Evangelink : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/code-quality/ca1716.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ ms.workload:
 
 The name of a namespace, type, or virtual or interface member matches a reserved keyword in a programming language.
 
-By default, this rule only looks at externally visible namespaces, types, and members, but this is [configurable](#configurability).
+By default, this rule only looks at externally visible namespaces, types, and members, but you can [configure visibility](#api-Surface) and [symbol kinds](#analyzed-symbol-kinds).
```
Comment (mavasani) [inline docs/code-quality/ca1716.md:31]:
```suggestion
By default, this rule only looks at externally visible namespaces, types, and members, but you can [configure visibility](#api-surface) and [symbol kinds](#analyzed-symbol-kinds).
```

**Comment by PRMerger17** [comment]
@Evangelink : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Mikejo5000** [comment]
#sign-off

**Comment by PRMerger20** [comment]
@Jak-MS : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**